### ISSUE TITLE
Process exited with code 2 on calls to model.fit() in DL4J- Had to uninstall Anaconda (Python)...

### ISSUE THREAD

**Issue by DevinCalado** [deeplearning4j/libnd4j#236]
Was suddenly running into issues when calling model.fit(dataset) - "Process exited with code 2"
Was happening on DL4J-examples as well.
No stack traces.

After discussion with @agibsonccc on gitter, I realized I had installed  Anaconda (Python 2.7) the day before - right when the issue started.

As a test, I uninstalled Anaconda from my machine.  That resolved the issues in DL4J.  There seems to be some conflict between DL4J and Anaconda.  

Issue is fixed for me.  Just posting this for further investigation.

**Comment by treo** [comment]
Can you add some more information about the crash? This sounds like it might be related to the MKL that is shipped with anaconda.

**Comment by DevinCalado** [comment]
@treo Hey.  I am running windows 8.1, intellij, dl4j and nd4j 0.4-rc3.10. 

CUDA and nd4j-native backends were experiencing the same issue.

Was receiving "Process finished with exit code 2" on any calls to model.fit(dataset).  No stack trace was ever in the console:

c.d.d.TrainDuplicateAdClassifier - Loading training data from csv...
c.d.d.TrainDuplicateAdClassifier - Normalizing training data...
c.d.d.TrainDuplicateAdClassifier - Build model....
c.d.d.TrainDuplicateAdClassifier - Fitting model....
Process finished with exit code 2

**Comment by raver119** [comment]
That wasn't conflict, because we've tested simple ops there, and they were
working. Also, cuda is affected too, but we're not using all there.
12 июня 2016 г. 0:43 пользователь "Devin Calado" <notifications@github.com>
написал:

**Comment by treo** [comment]
So I've downloaded anaconda, and it certainly is the packaged MKL that results in the error. What you tested on gitter was if libnd4j works when it doesn't invoke any blas related functions. The problem with mkl is that loading it works just fine, but when it initializes it fails.

And I'm pretty sure that @DevinCalado didn't actually have cuda running when he tried it.

The problem should be fixed with https://github.com/bytedeco/javacpp/commit/c731d1df8fda204a58a755bebc35b0d204aec587 but I didn't have time to try that out yet.

**Comment by DevinCalado** [comment]
@treo I'm pretty sure I did have Cuda running when I tried it.  I set the Cuda backend in the POM, and wasn't receiving any warnings or anything when running the training code.  Also, I have a tool for monitoring GPU performance and it definitely was running _something_ on the GPU...

I haven't had decent training performance on the GPU yet, so it is definitely possible that something related to Cuda isn't setup properly on my machine.

Thanks.
